### PR TITLE
update cloudflare known limitations

### PR DESCRIPTION
**Pull request by nataliejeremy**

### CODE REVIEW

File: source/docs/articles/sites/cloudflare-cdn-and-dns.md [hunk 1]
```diff
@@ -52,5 +52,28 @@ In Drupal, no modules are required to use CloudFlare, but there is an optional [
...
+X-Cf-Url: http://www.domain.com/sites/all/modules/admin_menu/admin_menu.js?nvql2t
+X-Cf-Status: 403
+Content-Transfer-Encoding: binary
+X-Cf-Max-Age: 7200
+X-Cf-Origin-Status: 403
+X-Cf-Cache-Status: EXPIRED
+Content-Type: text/html; charset=UTF-8
+X-Cf-Error: true
+```
+
```
Comment (rachelwhitton) [inline source/docs/articles/sites/cloudflare-cdn-and-dns.md]: @nataliejeremy should we add that the solution to this error is to disable the Rocket Loader feature in CloudFlare?

Comment (nataliejeremy) [inline source/docs/articles/sites/cloudflare-cdn-and-dns.md]: @rachelwhitton +1 if that's the best solution/suggestion

Comment (rachelwhitton) [inline source/docs/articles/sites/cloudflare-cdn-and-dns.md]: @natalie - Suggestion based on the [linked CF doc](https://support.cloudflare.com/hc/en-us/articles/200169456-Why-is-JavaScript-or-jQuery-not-working-on-my-site) - so the user doesn't need to load a new doc/site to find a solution. 

Looking again, we probably don't need to include the CF link at all - it just says to disable the feature or to report the bug to CF engineers.

I defer to you and @bmackinney for final edits -- :+1: from me!

Comment (nataliejeremy) [inline source/docs/articles/sites/cloudflare-cdn-and-dns.md]: Thanks--sounds good to me @rachelwhitton

**Pull request merged**